apps/loader/config.js

```diff
@@ -0,0 +1,3 @@
+const dropboxConfig = {
+  'APP_KEY': '6yibvsbuan5leo7',
```

Comment:
If this is a valuable API key, you can rebase it out.

Reply:
Thanks for the review. Resolved that. The `cloudUploadAPIs.md` covers more upon on how to get the APP_KEY configured. and it has been updated in the following commits.